docs/atomic-sign.1.md

```diff
@@ -0,0 +1,55 @@
+% ATOMIC(1) Atomic Man Pages
+% Brent Baude
+% August 2016
+# NAME
+atomic-sign- Create a signature for a local image
```

Comment:
This isn’t touching the local image (other than currently verifying that an image with such a name exists, but that shouldn’t happen I think)